python_modules/dagster/dagster/_utils/__init__.py

```diff
@@ -576,6 +576,18 @@ def find_free_port() -> int:
         return s.getsockname()[1]
 
 
+def is_port_in_use(host, port) -> bool:
+    sock = socket.socket(family=socket.AF_INET)
+    sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
only real change here

Reply:
maybe a comment "match the socket options uvicorn uses to bind"